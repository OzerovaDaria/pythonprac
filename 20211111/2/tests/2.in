qqqqqq
wwwwwwwwww
eeeeeee
rrrrrrrrrr
tttttt
yyyyyyyyyy
(), (), ()
(1,4), (2, 5), (3, 6)
(1,2), (4,5), (9,8)
