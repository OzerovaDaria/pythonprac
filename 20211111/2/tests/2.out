InvalidInput
InvalidInput
InvalidInput
InvalidInput
InvalidInput
InvalidInput
InvalidInput
Not a triangle
3.00
